src/main/kotlin/no/nav/klage/dokument/service/distribusjon/BrevMottakerDistribusjonService.kt

```diff
@@ -43,12 +47,23 @@ class BrevMottakerDistribusjonService(
             }
         }
 
-    private fun distribuerJournalpostTilMottaker(brevMottakerDistribusjon: BrevMottakerDistribusjon): BrevMottakerDistribusjon =
-        brevMottakerDistribusjon.copy(
+    private fun distribuerJournalpostTilMottaker(brevMottakerDistribusjon: BrevMottakerDistribusjon): BrevMottakerDistribusjon {
+        val dokumentType =
+            safClient.getJournalpostAsSystembruker(brevMottakerDistribusjon.journalpostId.value)?.tittel?.let {
+                DokumentType.fromBeskrivelse(
+                    it
+                )
+            }
+                ?: throw JournalpostNotFoundException("Journalpost med id ${brevMottakerDistribusjon.journalpostId.value} finnes ikke")
```

Comment:
Føles litt usikkert å basere seg på `tittel` for å finne `dokumentType`, men det kanskje er slik det må være?

Hvis `dokumentType` blir `null` (og vi skal kaste feil), så kan det vel også bety at `tittel` er `null`?